### DISCUSSION TITLE
unable to run jupyter notebook using the ms-python extension

### DISCUSSION THREAD

**Discussion by arunpatro** [coder/code-server#3508] [Q&A]
Since there are issues in the latest ms-python extension, I first turned auto-update off, and then downloaded the extension from https://open-vsx.org/extension/ms-python/python and ran `code-server --install-extension ms-python-release.vsix --force`. 

I get the following error when I am trying to run a cell in juypter notebook. =>

```
Activating Python 3.9.4 64-bit ('arun': conda) to run Jupyter failed with Error: Command failed: . activate arun && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /ssd_scratch/cvit/arunkumar/myconda/envs/arun/share/code-server/extensions/ms-python.python-2020.10.332292344/pythonFiles/pyvsc-run-isolated.py /ssd_scratch/cvit/arunkumar/myconda/envs/arun/share/code-server/extensions/ms-python.python-2020.10.332292344/pythonFiles/printEnvVariables.py bash: activate: No such file or directory .
```

**Comment by jsjoeio** [comment]
Hmm... I wonder what could be wrong 🤔 @oxy may be looking into this.
